ONSdigital/sdc-responses-dashboard
# dependabot alerts

# What and why?
This PR resolves a bunch of dependabot alerts. There were a ton, so it felt best to do them as one PR. Additionally, some of the dependabot alerts are more than a year old, and were trying to update the dependencies to versions that were themselves outdated.

# How to test?

- Run locally, and make sure the service works okay.
- Ensure that the following dependencies have been updated:

1. `jinja2`
2. `werkzeug`
3. `gunicorn`
4. `idna`
5. `black`
6. `certifi`
7. `requests`
8. `flask`
9. `es5-ext`
10. `semver`
11. `json5`

(The first eight are found in `pipfile.lock`, the last three in `package-lock.json` and/or `yarn.lock`).

# Jira
[Card](https://jira.ons.gov.uk/browse/RAS-1161)
